x-pack/legacy/plugins/siem/server/lib/detection_engine/notifications/get_signals_count.ts

```diff
@@ -4,63 +4,32 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import moment from 'moment';
-import { getNotificationResultsLink } from './utils';
 import { NotificationExecutorOptions } from './types';
-import { parseScheduleDates } from '../signals/utils';
 import { buildSignalsSearchQuery } from './build_signals_query';
 
-interface SignalsCountResults {
-  signalsCount: string;
-  resultsLink: string;
-}
-
 interface GetSignalsCount {
-  from: Date | string;
-  to: Date | string;
-  ruleAlertId: string;
+  from: string;
+  to: string;
   ruleId: string;
   index: string;
-  kibanaSiemAppUrl: string | undefined;
   callCluster: NotificationExecutorOptions['services']['callCluster'];
 }
 
 export const getSignalsCount = async ({
   from,
   to,
-  ruleAlertId,
   ruleId,
   index,
   callCluster,
-  kibanaSiemAppUrl = '',
-}: GetSignalsCount): Promise<SignalsCountResults> => {
-  const fromMoment = moment.isDate(from) ? moment(from) : parseScheduleDates(from);
-  const toMoment = moment.isDate(to) ? moment(to) : parseScheduleDates(to);
-
-  if (!fromMoment || !toMoment) {
-    throw new Error(`There was an issue with parsing ${from} or ${to} into Moment object`);
-  }
-
-  const fromInMs = fromMoment.format('x');
-  const toInMs = toMoment.format('x');
-
+}: GetSignalsCount): Promise<string> => {
   const query = buildSignalsSearchQuery({
     index,
     ruleId,
-    to: toInMs,
-    from: fromInMs,
+    to,
+    from,
   });
 
   const result = await callCluster('count', query);
-  const resultsLink = getNotificationResultsLink({
-    kibanaSiemAppUrl: `${kibanaSiemAppUrl}`,
-    id: ruleAlertId,
-    from: fromInMs,
-    to: toInMs,
-  });
 
-  return {
-    signalsCount: result.count,
-    resultsLink,
-  };
+  return result.count;
```

Comment:
Please type this return value, ideally in the originating call

Reply:
Type added